### PR TITLE
Spring Framework core Terminology

### PR DESCRIPTION
**Pull request by renukaThink**
Loose Coupling,Tightly Coupling,Dependencies and their types,launching spring simple program

### CODE REVIEW

File: day1-spring/learningspringframework/DependencyTypeEx/DependenceLauncherApp.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+
+@Component
+
```
Comment (yogesh-think) [inline day1-spring/learningspringframework/DependencyTypeEx/DependenceLauncherApp.java:14]: Please format a code changes before checked-in code changes.

File: day1-spring/learningspringframework/DependencyTypeEx/DependenceLauncherApp.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    public Vehicle(car c, bike b){
+
+        System.out.println("hey inside vehicle constructor");
+        this.b=b;
+        this.c=c;
+    }
+}
+@Component
```
Comment (yogesh-think) [inline day1-spring/learningspringframework/DependencyTypeEx/DependenceLauncherApp.java:50]: Create new file for new class

File: day1-spring/learningspringframework/DependencyTypeEx/DependenceLauncherApp.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+class bike{
+
+}
+
+
+
+
+@Configuration
+@ComponentScan("com.spring.learningspringframework.DependencyTypeEx")
```
Comment (yogesh-think) [inline day1-spring/learningspringframework/DependencyTypeEx/DependenceLauncherApp.java:63]: Use lower case for folder names

**Review by yogesh-think — APPROVED** [review]

File: day1-spring/learningspringframework/DependencyTypeEx/DependenceLauncherApp.java (hunk 3 above)
Comment (renukaThink) [reply day1-spring/learningspringframework/DependencyTypeEx/DependenceLauncherApp.java:63]: sure

**Pull request merged**